include/pwm.h

```diff
@@ -131,11 +131,11 @@ int32_t pwm_set_duty_cycle(struct pwm_desc *desc,
 int32_t pwm_get_duty_cycle(struct pwm_desc *desc,
 			   uint32_t *duty_cycle_ns);
 
-/* Set duty cycle of PWM generator device */
+/* Set the offset of PWM generator device */
```

Comment:
make sure to merge this commit with one of the previous commits that changes the same files